packages/graph/Makefile.toml

```diff
@@ -18,31 +12,66 @@ command = "yarn"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
 
 [tasks.recreate-db]
+run_task = { name = ["recreate-db-task", "migrate-up"] }
+
+[tasks.recreate-db-task]
+private = true
 category = "Deploy"
 description = "Recreates the database"
-command = "yarn"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
+command = "yarn"
 args = ["graph:recreate-db"]
 
 [tasks.migrate-up]
 category = "Deploy"
 description = "Runs all migrations"
-command = "yarn"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
+env_files = ["../.env"]
+command = "yarn"
 args = ["graph:migrate", "up"]
 
 [tasks.deployment-up]
 category = "Deploy"
 description = "Spins up the deployment environment"
-command = "docker-compose"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/deployment"
-args = ["--env-file", "../.env", "up", "-d"]
-dependencies = ["touch-environment"]
+command = "docker"
+args = ["compose", "--env-file", "../.env", "up", "--wait"]
 
 [tasks.deployment-down]
 category = "Deploy"
 description = "Tears down the deployment environment"
-command = "docker-compose"
 cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/deployment"
-args = ["--env-file", "../.env", "down"]
+command = "docker"
+args = ["compose", "--env-file", "../.env", "down"]
 
+[tasks.build-docker]
+category = "Deploy"
+description = "Spins up the Graph API as external service"
+cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/../../"
+env = { DOCKER_BUILDKIT = 1 }
+command = "yarn"
+args = ["external-services", "build", "graph"]
```

Comment:
Do we want to be adding ` "${@}"` to all of these `args`

Reply:
Yes, that's probably a good idea. I'd love to be able to also pass additional parameters from other tasks but I don't think this is possible

Reply:
Applied in 2b88cd0e39b010d1eb42fa3d6423a86c7f3a24c4